index.test.js

```diff
@@ -1,12 +1,21 @@
-const process = require("process");
```

Comment:
I know this can be pain, but would it possible to add in a separate PR the styling changes? It makes it quite hard to understand what are business logic changes and what's due to styling.

Reply:
Completely get that, the diff becomes quite ugly.
VS Code made these changes automatically, though, so it must be something with our IDE setups that isn't aligning. I have prettier and ES lint, and the CI did pass the linter test, so I guess we need to do some more configuration to make sure we're all running on the same settings.

Would you like me to manually revert the style changes (as far as I can at least)? 👀

Reply:
Update: I set the `.prettierrc`'s `"singleQuote"` to false and only formatted the test file using it. I'll make a PR after this to make the styling more consistent! 😄